docker/hq-compose.yml

```diff
@@ -33,14 +32,20 @@ services:
       TRAVIS_REPO_SLUG: "${TRAVIS_REPO_SLUG}"
     privileged: true  # allows mount inside container
     depends_on:
-      - postgres
-      - couch
-      - redis
-      - elasticsearch2
-      - elasticsearch5
-      - kafka
-      - zookeeper
-      - minio
+      postgres:
+        condition: service_started
```

Comment:
@kaapstorm why is this one `service_started` but in `hq-compose-runserver.yml` it is `service_health`?

Reply:
Great question! I think what happened is that when I added the health check for Postgres, I added it to `hq-compose-runserver.yml`, and forgot to add it to `hq-compose.yml`.